app/src/main/java/org/mozilla/tv/firefox/tabs/WebRenderTabsTray.kt

```diff
@@ -30,6 +33,23 @@ class WebRenderTabsTray @JvmOverloads constructor(
     internal val styling: TabsTrayStyling
     val tabsUseCases = context.webRenderComponents.tabsUseCases
 
+    private var focusDownId = -1
+    val pocketStateObserver = object : Observer<PocketViewModel.State> {
```

Comment:
If you only want to override `onNext` here, you can use something called a `Consumer`.  Behind the scenes it gets turned into an `Observer` with default implementations for the other three methods (Do you remember in the presentation when we subscribed but only passed one lambda?  That lambda was a Consumer.).  That would look like this.

```kotlin
    val pocketStateObserver = Consumer<PocketViewModel.State> { state ->
        focusDownId = when (state) {
            is PocketViewModel.State.Error -> R.id.megaTileTryAgainButton
            is PocketViewModel.State.Feed -> R.id.pocketVideoMegaTileView
            else -> R.id.tileContainer
        }
    }
```